Dockerfile

```
@@ -0,0 +1,69 @@
+##################################################
+#################### STAGE 1 #####################
+##################################################
+FROM rust:1-bookworm AS builder
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)